tests/cli/test_cli.py

```diff
@@ -1357,6 +1357,21 @@ def test_run(ctx):
         2 + 2 + 1, _run_command([cl, 'cat', remote_uuid])
     )  # 2 header lines, 1 stdout file, 1 stderr file, 1 item at bundle target root
 
+    # test metadata
+    uuid = _run_command(
+        [
+            cl,
+            "run",
+            "'for i in {1..60}; do sleep 1; done'",
+            "--request-memory",
+            "15m",
+            "--request-docker-image",
```

Comment:
I think you can remove `--request-docker-image`. It's automatically appended when you run `_run_command`.